rero_ils/modules/acq_orders/api.py

```diff
@@ -66,22 +76,23 @@ def create(cls, data, id_=None, delete_pid=False,
             data, id_, delete_pid, dbcommit, reindex, **kwargs)
         return record
 
+    def update(self, data, dbcommit=False, reindex=False):
+        """Update acq order record."""
+        self._acq_order_build_org_ref(data)
+        super(AcqOrder, self).update(data, dbcommit, reindex)
+        return self
+
     @classmethod
     def _acq_order_build_org_ref(cls, data):
         """Build $ref for the organisation of the acquisition order."""
-        library_pid = data.get('library', {}).get('pid')
-        if not library_pid:
-            library_pid = data.get('library').get(
-                '$ref').split('libraries/')[1]
-        org_pid = Library.get_record_by_pid(library_pid).organisation_pid
-        base_url = current_app.config.get('RERO_ILS_APP_BASE_URL')
-        url_api = '{base_url}/api/{doc_type}/{pid}'
-        org_ref = {
-            '$ref': url_api.format(
-                base_url=base_url,
-                doc_type='organisations',
-                pid=org_pid or cls.organisation_pid)
-        }
+        library = data.get('library', {})
+        library_pid = library.get('pid') or \
+            library.get('$ref').split('libraries/')[1]
+        org_ref = {'$ref': '{base_url}/api/{doc_type}/{pid}'.format(
```

Comment:
This code is repeated in almost every resource. a separate function in the `utils`?

Reply:
ok